<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="author" content="Gus Dunn" />
  <meta name="date" content="2014-12-18" />
  <title>RE java and snpEff</title>
  <style type="text/css">code{white-space: pre;}</style>
</head>
<body>
<div id="header">
<h1 class="title">RE java and snpEff</h1>
<h2 class="author">Gus Dunn</h2>
<h3 class="date">2014-12-18</h3>
</div>
<h1 id="section">2014-12-18</h1>
<h2 id="first-of-all">First of all:</h2>
<p>I am exceedingly proud of you! You totally rocked this email! You gave me like 99% of what I needed to think about this problem effectively without me having to ask a single question! I hope that you notice your improved computer literacy and allow yourself to feel some pride as well. You deserve it.</p>
<h2 id="so-on-to-the-snpeff-stuff">So on to the <code>snpEff</code> stuff:</h2>
<blockquote>
<p>when i run “which java” i get:</p>
</blockquote>
<pre><code>mariangelabonizzoni@dhcp-v106-224 17:31:49 ~:
which java
/usr/bin/java</code></pre>
<blockquote>
<p>when i call <code>java</code>, i get the help menu.</p>
</blockquote>
<blockquote>
<p>When i call <code>java -version</code>, i get the java version as:</p>
</blockquote>
<pre><code>mariangelabonizzoni@dhcp-v106-224 17:31:53 ~:
java -version
java version &quot;1.8.0_25&quot;
Java(TM) SE Runtime Environment (build 1.8.0_25-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.25-b02, mixed mode)</code></pre>
<blockquote>
<p>When i call <code>which snpEff</code>, i get:</p>
</blockquote>
<pre><code>which snpEff
/usr/local/bin/snpEff</code></pre>
<blockquote>
<p>snpEff is supposely a java program to be run through <code>java -jar snpEff.jar</code>, but when i do it, i always get the message:</p>
</blockquote>
<pre><code>mariangelabonizzoni@dhcp-v106-224 17:33:18 ~:
java -jar snpEff.jar
Error: Unable to access jarfile snpEff.jar</code></pre>
<blockquote>
<p>I tried to run this command with many different variants, like going into /user/local/bin or from /user/bin or using instead of sniper.jar the whole path to the program, but i always get the same message.</p>
<p>interestingly, when i simply type <code>snpEff</code>, i get the help menu. As follows</p>
</blockquote>
<pre><code>snpEff
Error: Missing command

snpEff version SnpEff 3.6c (build 2014-05-20), by Pablo Cingolani
Usage: snpEff [command] [options] [files]

Run &#39;java -jar snpEff.jar command&#39; for help on each specific command

Available commands:
        [eff]                        : Calculate effect of variants. Default: eff
(no command or &#39;eff&#39;).
        build                        : Build a SnpEff database.
        XXXXXXXXXXX
        -ud , -upDownStreamLen &lt;int&gt; : Set upstream downstream interval length
(in bases)</code></pre>
<h3 id="my-reply">My reply:</h3>
<p>Yes.</p>
<p>I am fairly certain that the correct way to call snpEff is to omit the <code>java -jar</code> call and simply call <code>snpEff</code> directly. The reason for this is that it has been installed in <code>/usr/local/bin</code> which is almost always in the user’s <code>$PATH</code> variable by default. The way that it is installed the <code>java -jar</code> portion is implied and the computer takes care of it behind the scenes.</p>
<p>The help text is an unfortunate confusion: for <strong>for sure</strong>. But again, I am quite proud that you basically figured this out without my help at all. Very nice work, Dr!</p>
<h2 id="the-genome-data-files">The genome data files:</h2>
<blockquote>
<p>the big problem that i have is that i have to build a genomic database because the organism that i want to study is not supported (it is an asian mosquito). i downloaded the required genome sequences and annotation file in GTF from vectorbase and i followed (i thik i did) the instruction in the “create database” from the snpEff menu page. but i keep getting the same message as: genome not found. see below. i tried i do not how amny different options to change the name of the files, to run the comand from different folders, to change the configuration file for snapped, i alwasy get the same message. like the genome file is not seen. it is driving me creasy!!!! any idea? I wrote to the snpEff developers, but thye have not had the curtesy to reply yet. any suggestion would be great!!! thanks</p>
</blockquote>
<pre><code>snpEff build -gtf22 /Volumes/Seagate_Exp_1/snpEff/data/AsinC2.1
java.lang.RuntimeException: Property:
&#39;/Volumes/Seagate_Exp_1/snpEff/data/AsinC2.1.genome&#39; not found
        at ca.mcgill.mcb.pcingola.interval.Genome.&lt;init&gt;(Genome.java:92)
        at ca.mcgill.mcb.pcingola.snpEffect.Config.readGenomeConfig(Config.java:513)
        at ca.mcgill.mcb.pcingola.snpEffect.Config.readConfig(Config.java:476)
        at ca.mcgill.mcb.pcingola.snpEffect.Config.init(Config.java:377)
        at ca.mcgill.mcb.pcingola.snpEffect.Config.&lt;init&gt;(Config.java:99)
        at
ca.mcgill.mcb.pcingola.snpEffect.commandLine.SnpEff.loadConfig(SnpEff.java:236)
        at
ca.mcgill.mcb.pcingola.snpEffect.commandLine.SnpEffCmdBuild.run(SnpEffCmdBuild.java:256)
        at ca.mcgill.mcb.pcingola.snpEffect.commandLine.SnpEff.run(SnpEff.java:685)
        at ca.mcgill.mcb.pcingola.snpEffect.commandLine.SnpEff.main(SnpEff.java:118)</code></pre>
<h3 id="my-reply-1">My reply:</h3>
<p>Ok. This is a bit more complicated. I am going to have to try looking through the <code>snpEff.config</code> file that they mention. It looks like this file is how you set up all the locations of things that you will need to organize to let <code>snpEff</code> know how to build its databases. Did you configure this file yet? If so, would you send me the file so I have a place to start?</p>
<p>Gus</p>
<div class="references">

</div>
</body>
</html>
