---
title: Preliminary LD results and VCF contents of master VCF file
author: Gus, Andrea
date: 2015-01-05
documentclass: scrartcl
classoption: letterpaper
geometry: margin=1in
toc: 1
graphics: 1
tags: [meeting, VCF file contents, ]
fontfamily: fourier
read: "+simple_tables+table_captions+footnotes+inline_notes+fenced_code_blocks+fenced_code_attributes+fancy_lists+definition_lists+superscript+subscript+tex_math_dollars"
...

# Summary #

## Discussed ##

- about half the SNP LD comparisons for `scaffold0` had one of the SNPs with MAF of 0.
- Andrea thought she filtered these out but SOME seem to still be there ovbiously.
- tracked one down in the VCF file (`tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.vcf`). See also the _"Example VCF line"_.
    - ref was `C` alt was `A` but not a single individual had the alt allele.
    - this should not be in this file.

## Resolution ##

- I collected a few lines from the VCF into a test file to send to Andrea
- Andrea will send this file along with a question to the developers (`VCFtools`? `PLINK`?) with the options she used for the filter to get an explanation.

## Follow up ##

2015-01-05 

- Andrea got back to me and it turns out that the `--min-allele`/`--max-allele` filter settings apply not to individuals' genotype but to the definition of the SNP at large and `Scaffold0-23159-C/A` WAS multi-allelic in the original data set.
- To filter snps that have no CURRENT individuals representing the alt allele other settings should be used and some but not all of her analyses will need to be re-run.
    - _correct filter:_ `--maf 0.00001`
- I can/will run the filter with the new settings on my version of the file myself.


# Example VCF line #

```
Scaffold0   23159   .   C   A   15.5    .   .   GT:PL:DP:SP:GQ  0/0:0,108,255:36:0:99   \
0/0:0,54,255:18:0:72    0/0:0,48,255:16:0:66    0/0:0,102,255:34:0:99   \
0/0:0,255,255:121:0:99  0/0:0,90,255:30:0:99    0/0:0,253,255:84:0:99   \
0/0:0,48,255:16:0:66    0/0:0,126,255:42:0:99   0/0:0,132,255:44:0:99   \
0/0:0,36,255:12:0:54    0/0:0,72,255:24:0:90    0/0:0,151,255:50:0:99   \
0/0:0,220,255:75:0:99   0/0:0,255,255:163:0:99  0/0:0,120,255:40:0:99   \
0/0:0,108,255:36:0:99   0/0:0,169,255:56:0:99   0/0:0,255,255:97:0:99   \
0/0:0,208,255:69:0:99   0/0:0,157,255:52:0:99   0/0:0,169,255:56:0:99   \
0/0:0,57,255:19:0:75    0/0:0,48,255:16:0:66    0/0:0,87,255:29:0:99    \
0/0:0,60,255:20:0:78    0/0:0,81,255:27:0:99    0/0:0,217,255:72:0:99   \
0/0:0,54,255:18:0:72    0/0:0,205,255:68:0:99   0/0:0,144,255:48:0:99   \
0/0:0,255,255:97:0:99   0/0:0,90,255:30:0:99    0/0:0,205,255:68:0:99   \
0/0:0,48,255:16:0:66    0/0:0,96,255:32:0:99    0/0:0,54,255:18:0:72    \
0/0:0,54,255:18:0:72    0/0:0,255,255:86:0:99   0/0:0,255,255:116:0:99  \
0/0:0,108,255:36:0:99   0/0:0,126,255:42:0:99   0/0:0,114,255:38:0:99   \
0/0:0,160,255:53:0:99   0/0:0,81,255:27:0:99    0/0:0,48,255:16:0:66    \
0/0:0,172,255:57:0:99   0/0:0,141,255:47:0:99   0/0:0,126,255:42:0:99   \
0/0:0,163,255:54:0:99   0/0:0,126,255:42:0:99   0/0:0,51,255:17:0:69    \
0/0:0,151,255:50:0:99
```