<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title>Daily Records</title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; }
code > span.dt { color: #902000; }
code > span.dv { color: #40a070; }
code > span.bn { color: #40a070; }
code > span.fl { color: #40a070; }
code > span.ch { color: #4070a0; }
code > span.st { color: #4070a0; }
code > span.co { color: #60a0b0; font-style: italic; }
code > span.ot { color: #007020; }
code > span.al { color: #ff0000; font-weight: bold; }
code > span.fu { color: #06287e; }
code > span.er { color: #ff0000; font-weight: bold; }
  </style>
</head>
<body>
<div id="header">
<h1 class="title">Daily Records</h1>
<h1 class="subtitle">Caccone PostDoc</h1>
<h3 class="date">December, 2014</h3>
</div>
<div id="TOC">
1
</div>
<h1 id="section"><span class="header-section-number">1</span> 2014-12-19</h1>
<h2 id="argot2-batch-size-reduction"><span class="header-section-number">1.1</span> Argot2 batch size reduction:</h2>
<pre class="tags"><code>tags = [argot2, ddRAD58, python, spartan]</code></pre>
<ul>
<li>dumping the whole proteome on the online server seems to have broken it.</li>
<li>I believe I will need to split the input up into around 3000 proteins per submission</li>
<li>this will need some python code (maybe added to <code>spartan</code>?)</li>
</ul>
<h3 id="problems"><span class="header-section-number">1.1.1</span> Problems:</h3>
<pre class="tags"><code>tags = [python, pip, virtualenvwrapper, py279, admin]</code></pre>
<ul>
<li>cant run ipython notebook bc of py279 from py278 issues.</li>
<li>must reinstall most/all python dependencies.</li>
</ul>
<hr />
<h1 id="section-1"><span class="header-section-number">2</span> 2014-12-21</h1>
<h2 id="argot2-batch-size-reduction-1"><span class="header-section-number">2.1</span> Argot2 batch size reduction:</h2>
<ul>
<li>see 2014-12-19</li>
<li>finished updating <code>py279</code> <code>python</code> requirements</li>
<li><code>ipython</code> seems to work again</li>
</ul>
<h3 id="completed"><span class="header-section-number">2.1.1</span> Completed:</h3>
<ul>
<li>writing filter functions for <code>blast</code> and <code>hmmer</code> outputs:
<ul>
<li><code>spartan/src/spartan/utils/blast/output.py</code>:
<ul>
<li><code>filter_for_argot(path, protein_names)</code></li>
</ul></li>
<li><code>spartan/src/spartan/utils/hmmer/output.py</code>:
<ul>
<li><code>filter_for_argot(path, protein_names)</code></li>
</ul></li>
</ul></li>
<li>created <code>sublime text 3</code> snippet to make new <code>ipython notebook</code> file with meta info and template.</li>
<li>wrote and tested split up logic:
<ul>
<li><code>spartan/src/spartan/utils/misc.py</code>:
<ul>
<li><code>split_stream(stream, divisor)</code></li>
</ul></li>
</ul></li>
</ul>
<h3 id="problems-1"><span class="header-section-number">2.1.2</span> Problems:</h3>
<ul>
<li>coding the split up logic</li>
</ul>
<h2 id="working-on"><span class="header-section-number">2.2</span> Working on:</h2>
<ul>
<li>split <code>blastp</code> and <code>hmmer</code> outputs by protein (~3000 per group)</li>
</ul>
<hr />
<h1 id="monday"><span class="header-section-number">3</span> 2014-12-22 (Monday)</h1>
<h2 id="argot2-batch-size-reduction-2"><span class="header-section-number">3.1</span> Argot2 batch size reduction:</h2>
<ul>
<li>see 2014-12-21</li>
<li>finished splitting</li>
<li>files were zipped</li>
<li>file pairs were submitted to Argot2</li>
</ul>
<h3 id="files"><span class="header-section-number">3.1.1</span> Files:</h3>
<pre><code>    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.0.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.1.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.2.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.3.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.4.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.5.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.6.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.7.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.8.hmmscan.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.9.hmmscan.zip

    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.0.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.1.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.2.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.3.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.4.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.5.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.6.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.7.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.8.blastp.zip
    Glossina-fuscipes-IAEA_PEPTIDES_GfusI1.union.9.blastp.zip</code></pre>
<h3 id="next"><span class="header-section-number">3.1.2</span> Next:</h3>
<ul>
<li>Collect the resulting analysis files into single repository.</li>
</ul>
<hr />
<h1 id="tuesday"><span class="header-section-number">4</span> 2014-12-23 (Tuesday)</h1>
<h2 id="meeting-with-dan"><span class="header-section-number">4.1</span> Meeting with Dan</h2>
<ul>
<li><a href="file:///home/gus/Dropbox/repos/git/markdown-docs/notes/meetings/dan-2014-12-23/dan-2014-12-23.md">markdown-docs/notes/meetings/dan-2014-12-23/dan-2014-12-23.md</a></li>
</ul>
<h2 id="argot2-batch-size-reduction-3"><span class="header-section-number">4.2</span> Argot2 batch size reduction:</h2>
<ul>
<li>see 2014-12-22</li>
<li><a href="http://localhost:8888/jupiter/notebooks/YALE/ddrad58/2014-12-23_create_argot2_functional_annotation_db_GfusI1.1_prerelease.ipynb">2014-12-23_create_argot2_functional_annotation_db_GfusI1.1_prerelease.ipynb</a></li>
</ul>
<hr />
<h1 id="wednesday"><span class="header-section-number">5</span> 2014-12-24 (Wednesday)</h1>
<h2 id="argot-batch-size-reduction"><span class="header-section-number">5.1</span> Argot batch size reduction</h2>
<ul>
<li>see 2014-12-23</li>
</ul>
<h3 id="storing-in-python-friendly-format-pandas.hdfstore"><span class="header-section-number">5.1.1</span> Storing in <code>Python</code>-friendly format (<code>pandas.HDFStore</code>)</h3>
<ul>
<li>had to install <code>pytables</code></li>
<li>test work well and data is stored</li>
<li>documentation: <a href="http://localhost:8888/jupiter/notebooks/YALE/ddrad58/2014-12-24_store_argot2_functional_annotation_db_GfusI1.1_prerelease_as_HDF5.ipynb">2014-12-24_store_argot2_functional_annotation_db_GfusI1.1_prerelease_as_HDF5.ipynb</a></li>
</ul>
<h3 id="file-location"><span class="header-section-number">5.1.2</span> File location</h3>
<pre><code>louise/data/genomes/glossina_fuscipes/annotations/ \
functional/GfusI1.1_pre/argot2_out/argot_functional_annotations_ts150.h5</code></pre>
<h3 id="project-completed"><span class="header-section-number">5.1.3</span> PROJECT COMPLETED</h3>
<p>“Argot batch size reduction” project is now considered completed.</p>
<h2 id="installing-pytables"><span class="header-section-number">5.2</span> Installing <code>PyTables</code></h2>
<p><strong>First attempt failed due to cryptic or at <em>least</em> slightly misleading error about <code>numpy</code> and <code>numexpr</code>:</strong></p>
<pre class="sourceCode bash"><code class="sourceCode bash">$ <span class="kw">pip</span> install git+https://github.com/PyTables/PyTables.git@v.3.1.1#egg=tables</code></pre>
<p><strong>Tried installing <code>numexpr</code> directly with:</strong></p>
<pre class="sourceCode bash"><code class="sourceCode bash">$ <span class="kw">pip</span> install numexpr</code></pre>
<p><strong>Tried <code>PyTables</code> again:</strong></p>
<pre class="sourceCode bash"><code class="sourceCode bash">$ <span class="kw">pip</span> install git+https://github.com/PyTables/PyTables.git@v.3.1.1#egg=tables</code></pre>
<h2 id="t-functional-annotations-of-genes-near-snps-of-interest"><span class="header-section-number">5.3</span> T: Functional Annotations of genes near SNPs of interest</h2>
<ul>
<li>Project start.</li>
</ul>
<hr />
<h1 id="friday"><span class="header-section-number">6</span> 2014-12-26 (Friday)</h1>
<h2 id="t-functional-annotations-of-genes-near-snps-of-interest-1"><span class="header-section-number">6.1</span> T: Functional Annotations of genes near SNPs of interest</h2>
<ul>
<li><code>[_]</code> write code to create table of functional annotation info, given gene-names and annotation database.
<ul>
<li><code>[_]</code> draft in ipython notebook</li>
<li><code>[_]</code> copy to <code>spartan</code></li>
</ul></li>
<li><code>[_]</code> write methods for the functional annotation paper section</li>
</ul>
<div class="references">

</div>
</body>
</html>
