<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="author" content="Gus Dunn" />
  <title>Daily Records</title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; }
code > span.dt { color: #902000; }
code > span.dv { color: #40a070; }
code > span.bn { color: #40a070; }
code > span.fl { color: #40a070; }
code > span.ch { color: #4070a0; }
code > span.st { color: #4070a0; }
code > span.co { color: #60a0b0; font-style: italic; }
code > span.ot { color: #007020; }
code > span.al { color: #ff0000; font-weight: bold; }
code > span.fu { color: #06287e; }
code > span.er { color: #ff0000; font-weight: bold; }
  </style>
  
  
  
</head>
<body>
<div id="header">
<h1 class="title">Daily Records</h1>
<h1 class="subtitle">Caccone PostDoc</h1>
<h2 class="author">Gus Dunn</h2>
<h3 class="date">February, 2015</h3>
</div>
<div id="TOC">
1
</div>
<hr />
<h1 id="sunday"><span class="header-section-number">1</span> 2015-02-01 (Sunday)</h1>
<h2 id="updating-maps-current-trap-locations"><span class="header-section-number">1.1</span> Updating maps: current trap locations</h2>
<h3 id="spartan-dev-gps-stuff"><span class="header-section-number">1.1.1</span> <code>spartan</code> dev: GPS stuff</h3>
<ul>
<li>writing autovivification version of <code>GPSCoordTree._grow_branch</code>.</li>
</ul>
<hr />
<h1 id="monday"><span class="header-section-number">2</span> 2015-02-02 (Monday)</h1>
<h2 id="updating-maps-current-trap-locations-1"><span class="header-section-number">2.1</span> Updating maps: current trap locations</h2>
<h3 id="spartan-dev-gps-stuff-1"><span class="header-section-number">2.1.1</span> <code>spartan</code> dev: GPS stuff</h3>
<ul>
<li>testing (<a href="file:///home/gus/Dropbox/repos/git/spartan/src/spartan/tests/test_utils_maps_gps.py">test_utils_maps_gps.py</a>):
<ul>
<li><code>[x]</code> <code>GPSCoordTree._grow_branch</code></li>
<li><code>[x]</code> <code>GPSCoordTree._get_subtree</code></li>
<li><code>[x]</code> <code>GPSCoordTree.mean</code></li>
</ul></li>
</ul>
<h2 id="creating-uganda-data-repo"><span class="header-section-number">2.2</span> Creating Uganda Data Repo</h2>
<ul>
<li><p><strong>local location:</strong></p>
<a href="file:///home/gus/Dropbox/uganda_data/data_repos/field_data">/home/gus/Dropbox/uganda_data/data_repos/field_data</a></li>
<li><p><strong>github address:</strong> <a href="https://github.com/CacconeLabYale/field_data.git" class="uri">https://github.com/CacconeLabYale/field_data.git</a></p></li>
</ul>
<hr />
<h1 id="tuesday"><span class="header-section-number">3</span> 2015-02-03 (Tuesday)</h1>
<h2 id="updating-maps-current-trap-locations-2"><span class="header-section-number">3.1</span> Updating maps: current trap locations</h2>
<ul>
<li>established comprehensive lists of village-ID-map and trap GPS locations for Uganda:
<ul>
<li><p><strong>village-ID-map:</strong></p>
<a href="file:///home/gus/Dropbox/uganda_data/data_repos/field_data/locations/names/uganda_village_id_map.csv">field_data/locations/names/uganda_village_id_map.csv</a></li>
<li><p><strong>trap GPS coords:</strong></p>
<p><a href="file:///home/gus/Dropbox/uganda_data/data_repos/field_data/locations/gps/traps/uganda_traps_gps.csv">field_data/locations/gps/traps/uganda_traps_gps.csv</a></p></li>
</ul></li>
</ul>
<hr />
<h1 id="wednesday"><span class="header-section-number">4</span> 2015-02-04 (Wednesday)</h1>
<h2 id="general-todo"><span class="header-section-number">4.1</span> General ToDo</h2>
<ul>
<li><code>[x]</code> email to confirm HR got my letter</li>
<li><code>[x]</code> meet with Gisella and Andrea [1130]
<ul>
<li><code>[X]</code> write up notes from meeting: <a href="file:///home/gus/Dropbox/repos/git/markdown-docs/notes/meetings/gisella_andrea_2015-02-04/gisella_andrea_2015-02-04.pdf">gisella_andrea_2015-02-04.pdf</a></li>
</ul></li>
<li><code>[x]</code> Talk to Ben E about the MAD idea.</li>
<li><code>[x]</code> create git repo for this paper</li>
<li><code>[ ]</code> begin development of the MAD idea</li>
<li><code>[X]</code> install LDna and R-studio</li>
<li><code>[X]</code> Located space to move the EPH <em>G. pallidipes</em> samples here to ESC with Rob</li>
</ul>
<h2 id="ddrad-stuff"><span class="header-section-number">4.2</span> ddRAD stuff</h2>
<h3 id="ld-detect-outlier-snp-pairs"><span class="header-section-number">4.2.1</span> LD: detect ‘outlier’ SNP-pairs</h3>
<ul>
<li><strong>I propose this method:</strong>
<ol style="list-style-type: decimal">
<li>for each distance group: collect <span class="math"><em>r</em><sup>2</sup></span> from <span class="math"> ±  ∼ 5</span> bp distance window
<ol style="list-style-type: lower-alpha">
<li>across genome</li>
<li>across scaffold</li>
</ol></li>
<li>calculate modified z-score (based on <em>median absolute deviation</em> rather than standard deviation: <strong>MAD is more robust than SD for HTS-type data</strong>)</li>
<li>flag any SNP-pair with <span class="math"><em>z</em> ≥ 3.5</span></li>
<li><em>possibly randomize data and calculate FDR to evaluate performance.</em>
<ol style="list-style-type: lower-alpha">
<li>perhaps vary the window from step 1 to use FDR to chose window that minimizes FDR.</li>
</ol></li>
</ol></li>
<li><strong>Ben E’s thoughts:</strong>
<ul>
<li>basically: this is probably a waste of time and energy
<ul>
<li>other more sophisticated methods have already been applied to this data with not much significance detected</li>
<li>why do we expect this work to yield better/more results?</li>
</ul></li>
</ul></li>
<li><strong>Gisella’s thoughts:</strong>
<ul>
<li>still should do it bc we will need it when we have more data</li>
</ul></li>
</ul>
<h3 id="install-ldna"><span class="header-section-number">4.2.2</span> Install LDna</h3>
<ul>
<li>github page: <a href="https://github.com/petrikemppainen/LDna">github.com/petrikemppainen/LDna</a></li>
<li>paper reference: <a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12369/abstract" class="uri">http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12369/abstract</a></li>
<li>installed <code>devtools</code> with <code>RStudio</code> gui: <strong>[successful]</strong></li>
<li><p>installed <code>LDna</code> with <code>devtools</code>: <strong>[successful]</strong></p>
<pre class="sourceCode R"><code class="sourceCode r">devtools::<span class="kw">install_github</span>(<span class="st">&quot;petrikemppainen/LDna&quot;</span>)</code></pre>
<ul>
<li>documentation: <a href="file:///home/gus/R/x86_64-unknown-linux-gnu-library/3.1/LDna/html/00Index.html">LDna/html/00Index.html</a></li>
</ul></li>
</ul>
<h3 id="ldna-notes"><span class="header-section-number">4.2.3</span> LDna notes</h3>
<ul>
<li><p>operates on:</p>
<blockquote>
<p>Lower diagonal matrix of pairwise LD values, <span class="math"><em>r</em><sup>2</sup></span> is strongly recommended</p>
</blockquote></li>
<li><p>the code below should generate what I want (<strong>I think</strong>):</p></li>
</ul>
<pre class="sourceCode bash"><code class="sourceCode bash"><span class="kw">plink</span> --vcf tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf \
<span class="kw">--allow-extra-chr</span> \
<span class="kw">--r2</span> bin  \
<span class="kw">--out</span> plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin</code></pre>
<h3 id="plink-run-for-ldna"><span class="header-section-number">4.2.4</span> PLINK run for LDna</h3>
<ul>
<li>ran the command below:</li>
</ul>
<pre><code>wd238 at compute-23-2 in ~GENOMES/glossina_fuscipes/annotations/SNPs (py278)
$ plink --vcf tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf \
&gt; --allow-extra-chr \
&gt; --r2 bin  \
&gt; --out plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin
</code></pre>
<ul>
<li>waiting for it to finish: <strong>[failed]</strong></li>
</ul>
<h3 id="louise-scratch-request-email"><span class="header-section-number">4.2.5</span> Louise Scratch Request Email</h3>
<blockquote>
<p><strong>netid:</strong> wd238 <strong>group:</strong> caccone <strong>anticipated usage:</strong></p>
<ul>
<li>~ 100G</li>
<li><p>&lt; 100 files <strong>purpose of usage:</strong></p></li>
<li>running <code>plink</code> <em>all_v_all</em> linkage disequilibrium calculations on ~40K SNPs</li>
<li>current attempt (documented below) gave a write failure which I think may be bc of some rather large tmp files generated during the process?</li>
<li><p>Does bumping up against our space quota have hard/immediate consequences like that?</p></li>
</ul>
<p><strong>error log:</strong></p>
<pre><code>wd238 at compute-23-2 in ~GENOMES/glossina_fuscipes/annotations/SNPs (py278)
$ plink --vcf tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf \
&gt; --allow-extra-chr \
&gt; --r2 bin  \
&gt; --out plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin
PLINK v1.90b2o 64-bit (25 Nov 2014)        https://www.cog-genomics.org/plink2
(C) 2005-2014 Shaun Purcell, Christopher Chang   GNU General Public License v3
Logging to plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin.log.
516842 MB RAM detected; reserving 258421 MB for main workspace.
--vcf: 73k variants complete.
plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin-temporary.bed
+
plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin-temporary.bim
+
plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin-temporary.fam
written.
73297 variants loaded from .bim file.
53 people (0 males, 0 females, 53 ambiguous) loaded from .fam.
Ambiguous sex IDs written to
plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin.nosex
.
Using up to 63 threads (change this with --threads).
Before main variant filters, 53 founders and 0 nonfounders present.
Calculating allele frequencies... done.
Total genotyping rate is 0.965098.
73297 variants and 53 people pass filters and QC.
Note: No phenotypes present.
--r2 square bin to
plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r2_bin.ld.bin
... done.

Error: File write failure.
</code></pre>
</blockquote>
<h3 id="github-repo-for-this-paper"><span class="header-section-number">4.2.6</span> Github repo for this paper</h3>
<ul>
<li>github page:<br /><a href="https://github.com/CacconeLabYale/gloria_soria_ddRAD_2015.git" class="uri">https://github.com/CacconeLabYale/gloria_soria_ddRAD_2015.git</a></li>
</ul>
<hr />
<h1 id="thursday"><span class="header-section-number">5</span> 2015-02-05 (Thursday)</h1>
<h2 id="mariangela-blacktie-install"><span class="header-section-number">5.1</span> Mariangela <code>blacktie</code> install</h2>
<ul>
<li>turns out i did NOT send Mariangela install instructions for the development branch</li>
<li>wrote quick install script for her to use and sent it</li>
</ul>
<h2 id="mad-idea"><span class="header-section-number">5.2</span> MAD idea</h2>
<ol style="list-style-type: decimal">
<li>for each group of SNPs <span class="math"><em>x</em></span> bp apart: collect <span class="math"><em>r</em><sup>2</sup></span> from <span class="math"> ±  ∼ 5</span> bp distance window around <span class="math"><em>x</em></span>:
<ol style="list-style-type: lower-alpha">
<li>across genome</li>
<li>across scaffold</li>
</ol></li>
<li>calculate modified z-score (based on <em>median absolute deviation</em> rather than standard deviation: <strong>MAD is more robust than SD for HTS-type data</strong>)</li>
<li>flag any SNP-pair with <span class="math"><em>z</em> ≥ 3.5</span></li>
<li>possibly randomize data and calculate FDR to evaluate performance.
<ol style="list-style-type: lower-alpha">
<li>perhaps vary the window-size from step 1 to use FDR to chose window-size that minimizes FDR.</li>
</ol></li>
</ol>
<h3 id="development"><span class="header-section-number">5.2.1</span> Development</h3>
<ul>
<li>ipython notebook: <a href="file:///home/gus/Dropbox/common/ipy_notebooks/YALE/ddrad58/2015-02-05_MAD_idea.ipynb">ddrad58/2015-02-05_MAD_idea.ipynb</a></li>
</ul>
<h2 id="g.-pallidipes"><span class="header-section-number">5.3</span> <em>G. pallidipes</em></h2>
<ul>
<li>Rob brought most to ESC this morning</li>
<li>doesn’t expect to need my truck for the rest</li>
</ul>
<hr />
<h1 id="friday"><span class="header-section-number">6</span> 2015-02-06 (Friday)</h1>
<h2 id="mad-idea-1"><span class="header-section-number">6.1</span> MAD idea</h2>
<h3 id="development-1"><span class="header-section-number">6.1.1</span> Development</h3>
<ul>
<li><p>LOTS of progress at ipython notebook: <a href="file:///home/gus/Dropbox/common/ipy_notebooks/YALE/ddrad58/2015-02-05_MAD_idea.ipynb">ddrad58/2015-02-05_MAD_idea.ipynb</a></p></li>
<li><p>See notes about plotting median and MAD with bootstrapped CIs near the bottom of above (commit dd7fe5da5733406edeaab6ce3c25b523b94552f2)</p></li>
</ul>
<hr />
<h1 id="monday-1"><span class="header-section-number">7</span> 2015-02-09 (Monday)</h1>
<p><strong>Goals:</strong></p>
<ul>
<li><code>[x]</code> Zimmer Workshop</li>
<li><code>[x]</code> Start Professional Development notebook</li>
<li><code>[x]</code> Find out how to process health reimbursement
<ul>
<li><code>[ ]</code> Get them ready for mailing
<ul>
<li><code>[x]</code> form</li>
<li><code>[ ]</code> receipts</li>
</ul></li>
<li><code>[X]</code> Assemble list of information I need from Sarah and send it to her</li>
</ul></li>
<li><code>[ ]</code> Progress on MAD idea</li>
<li><code>[ ]</code> Generate strategy for the week</li>
<li><code>[ ]</code> Sketch out abstract for Keystone? meeting</li>
<li><code>[ ]</code> find out if there is data available on tsetse control by area in Uganda
<ul>
<li>chemicals sold</li>
<li>etc</li>
</ul></li>
</ul>
<h2 id="health-reimbursement"><span class="header-section-number">7.1</span> Health reimbursement</h2>
<ul>
<li><a href="http://yalehealth.yale.edu/claims" class="uri">http://yalehealth.yale.edu/claims</a></li>
<li>Supplemental Claim form: <a href="http://yalehealth.yale.edu/sites/default/files/supplemental_claims_form.pdf" class="uri">http://yalehealth.yale.edu/sites/default/files/supplemental_claims_form.pdf</a></li>
<li>pharmacy claim form: <a href="http://yalehealth.yale.edu/sites/default/files/pharmacy_claim_form_restat_catamaran.pdf" class="uri">http://yalehealth.yale.edu/sites/default/files/pharmacy_claim_form_restat_catamaran.pdf</a></li>
</ul>
<h3 id="instructions-for-pharmacy-process"><span class="header-section-number">7.1.1</span> Instructions for pharmacy process</h3>
<ul>
<li>from website above</li>
</ul>
<blockquote>
<p>Include copies of prescription receipts showing the following information:</p>
<ul>
<li>Pharmacy Name, Address &amp; Phone Number</li>
<li>Patient Name</li>
<li>Prescription Number</li>
<li>Prescription Fill Date</li>
<li>Drug Name, Strength and NDC Code</li>
<li>Drug Quantity &amp; Days supply</li>
<li>Drug Cost</li>
<li>Amount Paid</li>
</ul>
<p>Please mail the Prescription Drug Claim Form and receipts to:</p>
<p>Restat<br />Patient Reimbursement<br />11900 W. Lake Park Drive<br />Milwaukee, WI 53224</p>
<p>Claims are honored for one year from the date of service. If you haven’t received a response to a claim within 60 days of filing, contact the Claims Department. You may call sooner to inquire if the claim has been received and is in process.</p>
</blockquote>
<hr />
<h1 id="tuesday-1"><span class="header-section-number">8</span> 2015-02-10 (Tuesday)</h1>
<p><strong>Goals:</strong></p>
<ul>
<li><code>[ ]</code> Get pharm claim ready for mailing
<ul>
<li><code>[x]</code> form</li>
<li><code>[ ]</code> receipts</li>
</ul></li>
<li><code>[ ]</code> Progress on MAD idea</li>
<li><code>[ ]</code> Generate strategy for the week</li>
<li><code>[ ]</code> Sketch out abstract for Keystone? meeting</li>
<li><code>[ ]</code> find out if there is data available on tsetse control by area in Uganda
<ul>
<li>chemicals sold</li>
<li>etc</li>
</ul></li>
<li><code>[ ]</code> figure out how to download zimmer files</li>
</ul>
<h2 id="health-reimbursement-1"><span class="header-section-number">8.1</span> Health reimbursement</h2>
<ul>
<li>printed form</li>
</ul>
<h2 id="met-with-postdoc-applicant-christina"><span class="header-section-number">8.2</span> Met with Postdoc applicant (Christina)</h2>
<ul>
<li>had lunch</li>
</ul>
<hr />
<h1 id="thursday-1"><span class="header-section-number">9</span> 2015-02-12 (Thursday)</h1>
<h2 id="health-reimbursement-2"><span class="header-section-number">9.1</span> Health reimbursement</h2>
<ul>
<li>Need Catherine’s member ID</li>
</ul>
<h2 id="mad-idea-2"><span class="header-section-number">9.2</span> MAD idea</h2>
<h3 id="development-2"><span class="header-section-number">9.2.1</span> Development</h3>
<ul>
<li><em>yesterday:</em>
<ul>
<li>bootstrap confidence intervals are functional</li>
<li>modified z-score is functional</li>
<li>used ggplot to provide nice figure showing rough progression of z-scored <span class="math"><em>r</em><sup>2</sup></span> through distance between snps</li>
</ul></li>
</ul>
<hr />
<h1 id="friday-1"><span class="header-section-number">10</span> 2015-02-13 (Friday)</h1>
<h2 id="g.-pallidipes-sample-catalog"><span class="header-section-number">10.1</span> <em>G. pallidipes</em> Sample catalog</h2>
<h3 id="summary-table"><span class="header-section-number">10.1.1</span> Summary table</h3>
<ul>
<li>data types:
<ul>
<li>location</li>
<li>symbols when present (<em>I assume you mean location symbol?</em>)</li>
<li>number of individuals</li>
<li>date range</li>
<li>is tissue?</li>
<li>is extraction?</li>
<li>analysis status</li>
</ul></li>
<li><p>will be done in <code>python</code> for increased flexibility by <strong>[Gus]</strong></p></li>
<li><p>notebook file: <a href="file:///home/gus/Dropbox/common/ipy_notebooks/YALE/pallidipes_kenya/2015-02-12_sample_catalog_summary.ipynb">2015-02-12_sample_catalog_summary.ipynb</a></p></li>
<li><p>Showed output to Gisella and she signed off on it after asking whether I could accommodate GEO COORDS when we get them.</p></li>
<li><p><strong>STATUS:</strong> <strong>[completed]</strong></p></li>
</ul>
<h3 id="primers-etc"><span class="header-section-number">10.1.2</span> Primers etc</h3>
<ul>
<li>RobH reports that he and KirstinD found many primers etc that were either designed for <em>G. pallidipes</em> or shown to work with it in the past.</li>
<li>testing on the primers will begin next week.</li>
</ul>
<h3 id="leg-extractions"><span class="header-section-number">10.1.3</span> Leg extractions</h3>
<ul>
<li>Rob did Xymogen extractions on 5 legs</li>
<li>NanoDrop indicates absorption at 260 but peaks look weird
<ul>
<li>probably bc the kit leaves EVERYTHING still in solution</li>
<li><code>[ ]</code> RobH will check with KirstinD about her extraction traces on <em>G. f. fuscipes</em> legs</li>
</ul></li>
</ul>
<h2 id="mad-idea-3"><span class="header-section-number">10.2</span> MAD idea</h2>
<h3 id="development-3"><span class="header-section-number">10.2.1</span> Development</h3>
<ul>
<li><strong>[completed]:</strong> functions to
<ul>
<li>update df with <code>distance_bin</code> and <code>mad_z</code></li>
<li>plot mad_z by bins</li>
</ul></li>
<li><strong>[to do]:</strong>
<ul>
<li>implement printing/saving snp-pairs that pass the z-filter</li>
</ul></li>
</ul>
<hr />
<h1 id="saturday"><span class="header-section-number">11</span> 2015-02-14 (Saturday)</h1>
<h2 id="mad-idea-4"><span class="header-section-number">11.1</span> MAD idea</h2>
<h3 id="development-4"><span class="header-section-number">11.1.1</span> Development</h3>
<ul>
<li>implement printing/saving snp-pairs that pass the z-filter</li>
</ul>
<hr />
<h1 id="monday-2"><span class="header-section-number">12</span> 2015-02-16 (Monday)</h1>
<h2 id="g.-f.-fuscipes-infection-summaries"><span class="header-section-number">12.1</span> <em>G. f. fuscipes</em>: infection summaries</h2>
<ul>
<li>ipython to get pivot table for infected flies
<ul>
<li>file: <a href="file:///home/gus/Dropbox/common/ipy_notebooks/YALE/g_f_fuscipes_general/2015-02-16_g_f_fuscipes_pandas_import.ipynb">2015-02-16_g_f_fuscipes_pandas_import.ipynb</a>
<ul>
<li>file of dumped pandas table of collection records for 2014 in hdf5 format:</li>
</ul></li>
</ul></li>
<li>add PCR detected fly statuses to main DB</li>
</ul>
<h2 id="g.-pallidipes-microsat-extraction-pilot"><span class="header-section-number">12.2</span> <em>G. pallidipes</em>: MicroSat extraction pilot</h2>
<ul>
<li>RobH spoke with KirstinD about strange NanoDrop traces:
<ul>
<li>KirstinD: hers looked the same, just used 260/280 values as presented</li>
<li>likely explanation is that the extraction kit is EXTREMELY dirty by design so the spec peaks are shifted around</li>
</ul></li>
<li>RobH is beginning PCRs with ITS primers (same that KirstinD is using on the <em>G. f. fuscipes</em>) today.</li>
<li>RobH is researching location names on the SerapA tubes (n ~ 6) bc GisellaC is not convinced the sheet SerapA included makes since.
<ul>
<li>RobH will google first</li>
<li>GusD will get GIS admin layers to search if google fails</li>
</ul></li>
<li><a href="https://github.com/xguse/blacktie/archive/v0.2.1.2-1.tar.gz">v0.2.1.2-1.tar.gz</a></li>
</ul>
<hr />
<h1 id="tuesday-2"><span class="header-section-number">13</span> 2015-02-17 (Tuesday)</h1>
<h2 id="meeting"><span class="header-section-number">13.1</span> meeting</h2>
<ul>
<li>escarpment Nguruman:
<ul>
<li>GisellaC try to get samples from extremes and in the middle</li>
</ul></li>
<li><code>[ ]</code> GusD send most recent version of protocol to BrianW</li>
</ul>
<h1 id="wednesday-1"><span class="header-section-number">14</span> 2015-02-18 (Wednesday)</h1>
<h2 id="g.-pallidipes-status-update-meeting"><span class="header-section-number">14.1</span> <em>G. pallidipes</em> status update meeting</h2>
<ul>
<li>GusD</li>
<li>RobH</li>
<li><p>KirstinD</p></li>
<li>extractions not working for a while with KirstinD</li>
<li>trouble shooting</li>
<li><p>KirstinD moving forward with extractions now</p></li>
</ul>
<hr />
<h1 id="saturday-1"><span class="header-section-number">15</span> 2015-02-21 (Saturday)</h1>
<p><strong>GOALS:</strong></p>
<ul>
<li><code>[worked on]</code> <em>G. f. fuscipes</em> infection summaries/maps for GisellaC meeting</li>
<li><code>[no work]</code> script for MariangelaB</li>
<li><code>[small work]</code> <span class="math"><em>r</em><sup>2</sup></span> per bin model</li>
</ul>
<h2 id="g.-f.-fuscipes-infection-summaries-1"><span class="header-section-number">15.1</span> <em>G. f. fuscipes</em>: infection summaries</h2>
<h3 id="converting-dates-to-yyyy-mm-dd"><span class="header-section-number">15.1.1</span> Converting dates to YYYY-MM-DD</h3>
<ul>
<li><a href="file:///home/gus/Documents/YalePostDoc/project_stuff/g_f_fucipes_uganda/collection_data/2014_spring_summer_from_rob.xlsx">2014_spring_summer_from_rob.xlsx</a>
<ul>
<li>added new function to <code>TsetseCheckout</code>:<br /><a href="file:///home/gus/Dropbox/repos/git/TsetseCheckout/TsetseCheckout/data/utils.py">TsetseCheckout/data/utils.py:convert_brit_dates_to_yyyy_mm_dd(string)</a></li>
<li>added new cell magic to ipython to send variable to clipboard:<br /><a href="https://gist.github.com/xguse/a01780ef22cfad8adaf9">clip_magic.py</a></li>
<li>used new function and the cell magic to copy, change, then paste back into spreadsheet.</li>
</ul></li>
<li><a href="file:///home/gus/Documents/YalePostDoc/project_stuff/g_f_fucipes_uganda/collection_data/2014_fall_for_pandas.xlsx">2014_fall_for_pandas.xlsx</a>
<ul>
<li>dates already fine</li>
</ul></li>
</ul>
<h3 id="adding-village-names-to-the-springsummer-excel-file"><span class="header-section-number">15.1.2</span> Adding Village names to the spring/summer excel file</h3>
<ul>
<li><p><strong>[COMPLETED]:</strong> 2015-02-22</p></li>
<li>created python hack to use the summary sheet info to generate the Village rows<br /><a href="/home/gus/Documents/YalePostDoc/project_stuff/g_f_fucipes_uganda/collection_data/traps_to_villages.py">YalePostDoc/project_stuff/g_f_fucipes_uganda/collection_data/traps_to_villages.py</a>
<ul>
<li>summary sheets:<br /><a href="file:///home/gus/Dropbox/uganda_data/2014_Collection_Sheets_Spring-Summer/2014_full_surveyreport_20140820/summary%20survey%20data.xlsx">2014_full_surveyreport_20140820/summary survey data.xlsx</a></li>
</ul></li>
</ul>
<h3 id="alert-errors-detected-in-fly-name-code-combinations"><span class="header-section-number">15.1.3</span> ALERT: errors detected in fly name code combinations</h3>
<ul>
<li>during this process i detected instances where the fly number code combinations (example: <code>OLW-14 038</code>) were <strong>NOT</strong> correct!</li>
<li>the following IDs illustrate this:
<ul>
<li><code>OLO-14 033</code> is Olobo</li>
<li><code>OLO-14 034</code> is Olobo</li>
<li><code>OLW-14 035</code> is Olwi</li>
<li><code>OLW-14 036</code> is Olwi</li>
<li><code>OLW-14 037</code> is Olobo</li>
<li><code>OLW-14 038</code> is Olobo</li>
</ul></li>
<li><p>additionally, the <code>Dissection Data-Kole</code> sheet has <strong>ALL</strong> fly IDs starting <code>KO</code> regardless of the source village.</p></li>
<li><p><strong>RECOMEND <em>NOT</em> DEPENDING ON FLY ID FOR VILLAGE SOURCE!</strong></p></li>
</ul>
<hr />
<h1 id="sunday-1"><span class="header-section-number">16</span> 2015-02-22 (Sunday)</h1>
<p><strong>GOALS:</strong></p>
<ul>
<li><code>[worked on]</code> <em>G. f. fuscipes</em> infection summaries/maps for GisellaC meeting</li>
<li><code>[none]</code> script for MariangelaB</li>
<li><code>[none]</code> <span class="math"><em>r</em><sup>2</sup></span> per bin model</li>
</ul>
<h2 id="g.-f.-fuscipes-infection-summaries-2"><span class="header-section-number">16.1</span> <em>G. f. fuscipes</em>: infection summaries</h2>
<h3 id="hdf5-import-and-data-cleaning"><span class="header-section-number">16.1.1</span> HDF5 import and data cleaning</h3>
<ul>
<li><p>standardized the spreadsheet column titles by hand to allow import and correct dataframe referencing</p></li>
<li><p>file: <a href="file:///home/gus/Dropbox/common/ipy_notebooks/YALE/g_f_fuscipes_general/2015-02-16_g_f_fuscipes_pandas_import.ipynb">2015-02-16_g_f_fuscipes_pandas_import.ipynb</a></p></li>
<li><code>recode_villages(df, map_func=map_func)</code>:
<ul>
<li>renaming villages to letter codes</li>
<li><strong>[degenerate names discovered]</strong> and accommodated in<br /><a href="file:///home/gus/Dropbox/uganda_data/data_repos/field_data/locations/names/uganda_village_id_map.csv">uganda_village_id_map.csv</a> by mapping the letter code to more than one long form:
<ul>
<li>AKAYODEBE vs AKAYO-DEBE</li>
</ul></li>
<li>corrected misspellings of
<ul>
<li>“Orubakulemi” from “Orubakulem”</li>
<li>“JIAKO” from “JAIKO”</li>
</ul></li>
</ul></li>
<li><code>recode_positives(df)</code>:
<ul>
<li>recode <code>prob</code>, <code>midgut</code>, <code>sal.gland</code> as <code>0</code> or <code>1</code>.</li>
<li><strong>[NOTE]</strong> this will change to a trivalent state (class <code>Tristate</code>) soon</li>
</ul></li>
<li><code>recode_tenerals(df)</code>
<ul>
<li>implemented but needs conversion to <code>Tristate</code></li>
</ul></li>
<li><code>recode_dead(df)</code>
<ul>
<li>implemented but needs conversion to <code>Tristate</code></li>
</ul></li>
<li><code>add_infection_state_col(df)</code>
<ul>
<li>implemented but failing to actually alter the dataframe</li>
</ul></li>
<li><code>spartan.utils.misc.Tristate</code>
<ul>
<li>implements three state logic that <em>mostly</em> supports normal boolean arithmetic (just ignoring the <code>None</code> state)</li>
</ul></li>
</ul>
<h1 id="monday-3"><span class="header-section-number">17</span> 2015-02-23 (Monday)</h1>
<p><strong>GOALS:</strong></p>
<ul>
<li><code>[ ]</code> <em>G. f. fuscipes</em> infection summaries/maps for GisellaC meeting</li>
<li><code>[ ]</code> script for MariangelaB</li>
<li><code>[ ]</code> <span class="math"><em>r</em><sup>2</sup></span> per bin model</li>
</ul>
<h2 id="g.-f.-fuscipes-infection-summaries-3"><span class="header-section-number">17.1</span> <em>G. f. fuscipes</em>: infection summaries</h2>
<h3 id="hdf5-import-and-data-cleaning-1"><span class="header-section-number">17.1.1</span> HDF5 import and data cleaning</h3>
<ul>
<li><code>spartan.utils.misc.Tristate</code>
<ul>
<li>I found an existing “Tribool” class on github and forked it:<br /><a href="https://github.com/xguse/python_tribool" class="uri">https://github.com/xguse/python_tribool</a></li>
<li>it did not support boolean arithmetic but was much more sophisticated in all other ways.</li>
<li>I added support for boolean addition but will also add *, -, and / before writing the tests and submitting a pull request to upstream.</li>
<li>I am now using <code>Tribool</code> instead of <code>Tristate</code></li>
</ul></li>
<li>running into serious hashable issues <code>df.midgut.unique()</code> throws <code>__nonezero__</code>’s <code>ValueError</code>.
<ul>
<li>possible solutions:
<ul>
<li>override <code>__new__</code> might allow me to mimic the “always the same mem address” behavior of <code>True</code> etc?</li>
<li>Look into implementation of Factories in Python</li>
<li>perhaps a hint in behavior/class code for <code>np.NaN</code>?</li>
<li><strong>[best bet]</strong> use <code>enum</code> class</li>
</ul></li>
</ul></li>
<li>looking for more fertile ground to cover while I think</li>
</ul>
<div class="references">

</div>
</body>
</html>
