<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="author" content="Gus Dunn" />
  <meta name="date" content="2014-12-29" />
  <title>Gloria-Soria (ddRAD58)</title>
  <style type="text/css">code{white-space: pre;}</style>
  
  
</head>
<body>
<div id="header">
<h1 class="title">Gloria-Soria (ddRAD58)</h1>
<h2 class="author">Gus Dunn</h2>
<h3 class="date">2014-12-29</h3>
</div>
<div id="TOC">
1
</div>
<h1 id="functional-annotations"><span class="header-section-number">1</span> 1: Functional Annotations</h1>
<h2 id="sequences-used"><span class="header-section-number">1.1</span> 1.1: Sequences Used</h2>
<p><strong><em>Glossina fuscipes fuscipes:</em></strong> All putative peptides annotated for <em>G. f. fuscipes</em> in the GfusI1.1 gene-build were obtained from <a href="link_address">VectorBase</a> <span class="citation">[1]</span></p>
<p><strong>Other:</strong> The sequences used to compare the <em>G. f. fuscipes</em> peptides against well known/annotated sequences were obtained from UNIPROT/SwissProt <span class="citation">[2]</span> (used with <code>blastp</code>) and PFAM <span class="citation">[3]</span> (used with <code>hmmscan</code>) as required by <a href="link_address">ARGOT2</a> <span class="citation">[4–6]</span>.</p>
<h2 id="argot2-analysis"><span class="header-section-number">1.2</span> 1.2: Argot2 Analysis</h2>
<p>The <code>blastp</code> and <code>hmmscan</code> results submitted to ARGOT2 were obtained by performing local searches on the <em>G. f. fuscipes</em> peptides against the UNIPROT peptide database (obtained on 2014-09-08) and the hidden Markov models (HMM) of the combined protein-domain sets om the Pfam databases (Pfam-A and Pfam-B: obtained on 2014-09-08), respectively. Settings used were as dictated by the ARGOT2 site.</p>
<!-- Settings used were as dictated by the ARGOT2 site and summarized here.

For `blastp`: 

~~~~~~~~ {.bash}
blastp -outfmt "6 qseqid sseqid evalue " -query GFUSI1_1_PEPTIDES \
    -db UNIPROT_PEPTIDES -out OUTPUT_FILE
~~~~~~~~~~~~~~~~~~~

For `hmmscan`:

~~~~~~~~ {.bash}
hmmscan --tblout OUTPUT_FILE PFAMA_AND_PFAMB_DOMAINS GFUSI1_1_PEPTIDES
~~~~~~~~~~~~~~~~~~~ -->
<p>The <code>blastp</code> and <code>hmmscan</code> results were uploaded to ARGOT2 servers for analysis after being split into 10 groups (roughly 2330 peptides per group) to prevent overloading the remote ARGOT2 cluster. The functional annotations were then downloaded and joined back together.</p>
<h1 id="linkage-analysis"><span class="header-section-number">2</span> 2: Linkage Analysis</h1>
<h2 id="source-of-snps"><span class="header-section-number">2.1</span> 2.1: Source of SNPs</h2>
<p>SNPs were obtained as described earlier in this manuscript.</p>
<h2 id="linkage-measurements"><span class="header-section-number">2.2</span> 2.2: Linkage measurements</h2>
<!-- ~~~~~~~~ {.bash}
plink --vcf tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf \
--allow-extra-chr \
--r gz with-freqs dprime \
--out plink_out/tsetseFINAL_14Oct2014_f2_53.recode.renamed_scaffolds.maf0_05.vcf/ld/r_none_freqs_dprime
~~~~~~~~~~~~~~~~~~~ -->
<p>Plink version 1.9 <span class="citation">[7]</span> was used to calculate pairwise linkage disequilibrium (LD) as <span class="math"><em>r</em></span> for all SNP-pairs located on common supercontigs. The <code>--allow-extra-chr</code> option was required to handle the number of supercontigs. Unless stated otherwise, all subsequent analysis pertaining to LD used <span class="math"><em>r</em><sup>2</sup></span>.</p>
<h2 id="ld-based-filtering-of-snp-pairs"><span class="header-section-number">2.3</span> 2.3: LD-based filtering of SNP-pairs</h2>
<p>The LD values of SNP-pairs were compared after binning SNP-pairs by base pair separation to control for unknown rates of recombination in <em>G. f. fuscipes</em>. Bin length was set at 50 bp with the lower bound inclusive and higher bound exclusive. In other words the bins were defined as <span class="math">[<em>i</em>, <em>i</em> + 49)</span> where <span class="math"><em>i</em> ∈ {1, 1(50), 2(50), 3(50)...<em>n</em>(50)}</span> and <span class="math"><em>n</em></span> is a positive integer.</p>
<p>To identify SNP-pairs for further investigation in an arbitrary set of binned SNP-pairs, we needed to assign probabilities to each SNP-pair in the bin. The distributions of binned SNP-pairs are bounded by 0 and 1 and do not appear to be Normal in shape. Additionally, in many cases, the data appear to exhibit peaks at both the lower and upper <span class="math"><em>r</em><sup>2</sup></span> range. This suggested that the data may be modeled well using the probability density function (PDF) of the Beta distribution:</p>
<p><br /><span class="math">$$f(x;\alpha,\beta) = \frac{x^{\alpha-1}(1-x)^{\beta-1}} {\mathrm{B}(\alpha,\beta)}$$</span><br /></p>
<p>with shape parameters <span class="math"><em>α</em></span> and <span class="math"><em>β</em></span> and where <span class="math"><em>x</em></span> represents an observed value generated by the distribution and <span class="math"><em>B</em>()</span> is the Beta function <strong>{{CITE_ME}}</strong>.</p>
<p>The Beta’s cumulative distribution function (CDF)</p>
<p><br /><span class="math">$$F(x;\alpha,\beta) = \frac{\mathrm{B}(x;\alpha,\beta)}{\mathrm{B}(\alpha,\beta)} $$</span><br /></p>
<p>can be used to describe the probability that a binned SNP-pair will be observed to have an <span class="math"><em>r</em><sup>2</sup> ≤ <em>x</em></span>. It follows that <span class="math">1 − <em>C</em><em>D</em><em>F</em></span> represents the probability of observing a more extreme value.</p>
<p>For each set of binned <span class="math"><em>r</em><sup>2</sup></span> values, the SNP-pairs deemed worthy of further investigation were defined as those where <span class="math">1 − <em>C</em><em>D</em><em>F</em> ≤ 0.01</span> after Benjamini-Hochberg (BH) correction for multiple testing <span class="citation">[8]</span>.</p>
<h3 id="scaling-of-binned-r2-distributions"><span class="header-section-number">2.3.1</span> 2.3.1: Scaling of binned <span class="math"><em>r</em><sup>2</sup></span> distributions</h3>
<p>The Beta distribution is bounded on the non-inclusive interval between 0 and 1. However, there are data in each bin that may have been assigned values of exactly 0 or 1. It is likely that these values are not truly 0 or 1 in the discrete binary since that a coin-flip is <em>either</em> heads or tails. Therefore, the all data for each bin were scaled according to the following scheme</p>
<p><br /><span class="math">((<em>x</em><sub><em>i</em></sub> − 0.5) ⋅ <em>θ</em>) + 0.5</span><br /></p>
<p>in order to symmetrically shrink the distribution of values to fit within the open interval <span class="math">(0, 1)</span>. In the scheme above, let <span class="math"><em>x</em><sub><em>i</em></sub></span> stand for the <span class="math"><em>r</em><sup>2</sup></span> of each SNP-pair in an arbitrary bin set and <span class="math"><em>θ</em></span> stand for the scaling factor. The relevant results in this paper used <span class="math"><em>θ</em> = 0.999</span>.</p>
<h3 id="bayesian-parameter-estimation-using-the-binned-r2-distributions"><span class="header-section-number">2.3.2</span> 2.3.2: Bayesian parameter estimation using the binned <span class="math"><em>r</em><sup>2</sup></span> distributions</h3>
<p>In order to use the CDF of the Beta distribution to assign significances to each observed SNP-pair in a bin, it is necessary to learn the distribution parameters (<span class="math"><em>α</em></span> and <span class="math"><em>β</em></span>) for each bin given the specific data in each bin. For this we used custom python code that made heavy use of the following third-party data analysis modules: Pandas <span class="citation">[9]</span>, NumPy <span class="citation">[10]</span>, SciPy <span class="citation">[11]</span>, pyMC <span class="citation">[12]</span>, and StatsModels <span class="citation">[13]</span>.</p>
<p>We used pyMC to build the model of the Beta distribution and use it to exploit the bin-specific data to estimate the bin-specific values for the <span class="math"><em>α</em></span> and <span class="math"><em>β</em></span> parameters of the model [Figure fig_betamod]. The values of <span class="math"><em>α</em></span> and <span class="math"><em>β</em></span> were modeled with separate Uniform prior distributions from 0.01 to 10. This model topology was used to create pyMC “model” objects and initialized with the <span class="math"><em>r</em><sup>2</sup></span> data from each bin. The parameters of each Beta distribution were then estimated by <em>maximum a posteriori</em> (MAP) and used to calculate the <span class="math">1 − <em>C</em><em>D</em><em>F</em></span> for each SNP-pair in each bin <span class="citation">[10,11]</span>. The <span class="math">1 − <em>C</em><em>D</em><em>F</em></span> values were then BH corrected by bin and filtered with a threshold of <span class="math">(1 − <em>C</em><em>D</em><em>F</em>)<sub><em>B</em><em>H</em></sub> ≤ 0.01</span> using statsmodels <span class="citation">[8]</span>.</p>
<div id="fig_betamod">
<div class="figure">
<img src="../figures/bin_MAP_model.png" alt="[fig_betamod]Network representation of the LD Beta model: Ovals represent modeled probability distributions. Circles represent learned parameters. Grey shading indicates use of observed data. " /><p class="caption"><strong>[fig_betamod]Network representation of the LD Beta model:</strong> Ovals represent modeled probability distributions. Circles represent learned parameters. Grey shading indicates use of observed data. </p>
</div>
</div>
<!-- ################################################################## -->
<p><!-- ################################################################## --></p>
<div class="references">
<h1>3: Bibliography</h1>
<p>1. Giraldo-Calderon GI, Emrich SJ, MacCallum RM, Maslen G, Dialynas E, Topalis P, et al. VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases. Nucleic Acids Research. 2014;43: D707–D713. doi:<a href="http://dx.doi.org/10.1093/nar/gku1117">10.1093/nar/gku1117</a></p>
<p>2. Boeckmann B, Blatter M-C, Famiglietti L, Hinz U, Lane L, Roechert B, et al. Protein variety and functional diversity: Swiss-Prot annotation in its biological context. Comptes rendus biologies. 2005;328: 882–99. doi:<a href="http://dx.doi.org/10.1016/j.crvi.2005.06.001">10.1016/j.crvi.2005.06.001</a></p>
<p>3. Finn RD, Bateman A, Clements J, Coggill P, Eberhardt RY, Eddy SR, et al. Pfam: the protein families database. Nucleic acids research. 2014;42: D222–30. doi:<a href="http://dx.doi.org/10.1093/nar/gkt1223">10.1093/nar/gkt1223</a></p>
<p>4. Radivojac P, Clark WT, Oron TR, Schnoes AM, Wittkop T, Sokolov A, et al. A large-scale evaluation of computational protein function prediction. Nature methods. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved. 2013;10: 221–7. doi:<a href="http://dx.doi.org/10.1038/nmeth.2340">10.1038/nmeth.2340</a></p>
<p>5. Falda M, Toppo S, Pescarolo A, Lavezzo E, <span>Di Camillo</span> B, Facchinetti A, et al. Argot2: a large scale function prediction tool relying on semantic similarity of weighted Gene Ontology terms. BMC bioinformatics. BioMed Central Ltd; 2012;13 Suppl 4: S14. doi:<a href="http://dx.doi.org/10.1186/1471-2105-13-S4-S14">10.1186/1471-2105-13-S4-S14</a></p>
<p>6. Gillis J, Pavlidis P. Characterizing the state of the art in the computational assignment of gene function: lessons from the first critical assessment of functional annotation (CAFA). BMC Bioinformatics. BioMed Central Ltd; 2013;14: S15. doi:<a href="http://dx.doi.org/10.1186/1471-2105-14-S3-S15">10.1186/1471-2105-14-S3-S15</a></p>
<p>7. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaScience. BioMed Central Ltd; 2015;4: 7. doi:<a href="http://dx.doi.org/10.1186/s13742-015-0047-8">10.1186/s13742-015-0047-8</a></p>
<p>8. Benjamini Y, Hochberg Y. Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. Journal of the Royal Statistical Society Series B (Methodological). Wiley for the Royal Statistical Society; 1995;57: pp. 289–300. Available: <a href="http://www.jstor.org/stable/2346101" class="uri">http://www.jstor.org/stable/2346101</a></p>
<p>9. McKinney W. Data Structures for Statistical Computing in Python. In: Walt S van der, Millman J, editors. Proceedings of the 9th python in science conference. 2010. pp. 51–56. </p>
<p>10. <span>Van Der Walt</span> S, Colbert SC, Varoquaux G. The NumPy array: A structure for efficient numerical computation. Computing in Science and Engineering. 2011;13: 22–30. doi:<a href="http://dx.doi.org/10.1109/MCSE.2011.37">10.1109/MCSE.2011.37</a></p>
<p>11. Jones E, Oliphant T, Peterson P, Others. SciPy: Open source scientific tools for Python [Internet]. 2001. Available: <a href="http://www.scipy.org/" class="uri">http://www.scipy.org/</a></p>
<p>12. Patil A, Huard D, Fonnesbeck CJ. PyMC: Bayesian Stochastic Modelling in Python. Journal of statistical software. 2010;35: 1–81. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3097064\&amp;tool=pmcentrez\&amp;rendertype=abstract" class="uri">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3097064\&amp;tool=pmcentrez\&amp;rendertype=abstract</a></p>
<p>13. Statsmodels-development-team. StatsModels: Statistics in Python (v0.6.1) [Internet]. Available: <a href="http://statsmodels.sourceforge.net/stable/" class="uri">http://statsmodels.sourceforge.net/stable/</a></p>
</div>
</body>
</html>
